src/main/java/org/zalando/jackson/datatype/money/MoneyModule.java

```diff
@@ -11,11 +12,13 @@
 import javax.money.CurrencyUnit;
 import javax.money.MonetaryAmount;
 import java.util.Currency;
+import javax.money.NumberValue;
 
 import static com.fasterxml.jackson.core.util.VersionUtil.mavenVersionFor;
 
 public final class MoneyModule extends Module {
 
+    private final JsonSerializer<NumberValue> numberValueSerializer;
```

Comment:
This will not allow something like `integer + scale` since that requires two fields.